CMakeLists.txt

```diff
@@ -407,9 +407,13 @@ set(ITK_DATA_ROOT ${ITK_SOURCE_DIR}/Testing/Data)
 # Location of ITK Example Data.
 set(ITK_EXAMPLE_DATA_ROOT "${ITK_SOURCE_DIR}/Examples/Data")
 
-#This flag is used in particular, to enable some tests that require large memory to run.
-# This probably can be discovered at configuration time by CMake. (Check with CMake developers).
-set(ITK_COMPUTER_MEMORY_SIZE 1 CACHE STRING "Provide here the size of your RAM in gibibytes")
+# This flag is used in particular, to enable some tests that require large memory to run.
+# Allow for tests that may use upto 80% of the computer memory to be used for testing.
```

Comment:
We should also document that tests which use a significant amount of memory, also need to add the test property "RESOURCE_LOCK MEMORY_SIZE". This restricts test execution so that only one memory intensive test is run at once.

Reply:
I think that tests with _large_ in their name are good candidates for checking, as well as those with label `RUNS_LONG`.

Reply:
This was already instrumented.  I can not find any cases where this is not true.